lib/matplotlib/tests/test_dates.py

```diff
@@ -474,3 +474,10 @@ def test_tz_utc():
 def test_num2timedelta(x, tdelta):
     dt = mdates.num2timedelta(x)
     assert dt == tdelta
+
+
+def test_timedelta_ordinalf():
```

Comment:
I have a religious objection to this test. I don't believe that the private interface should be explicitly tested like this (this is in fact *not possible* in less permissive languages). You can see the various [answers on this SO thread](https://stackoverflow.com/questions/105007/should-i-test-private-methods-or-only-public-ones) for a list of reasons why not.

I would suggest implementing some part of the public interface that uses this `_tdelta_to_ordinalf(dt)` function and writing tests for *that*.

Reply:
I'll try and put a proper plotting test with timedeltas in.